### PR TITLE
mtpcopy: Add version 0.3.0

### PR DESCRIPTION
**Pull request by animegamer4422**
Mtpcopy is a Windows CLI tool for copying files from/to a portable device via MTP.

Closes #XXXX

Relates to #XXXX

- [ ] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by issaclin32** [comment]
* Please put items in the order of `version`, `description`, `homepage`, `license`. This is the convention for official buckets.

* Your .json file has a weird character `U+00A0` (NO-BREAK SPACE) in the front of each line, which can cause CI fail and some unexpected situations.

* [MTPCopy uses MIT license](https://github.com/kzmi/mtpcopy/blob/main/LICENSE). So it should be `"license": "MIT"`. For other licenses, check [SPDX license identifiers](https://spdx.org/licenses/).

* The app is bulit in **64-bit**, so please put `url` and `hash` like:
```
"architecture": {
    "64bit": {
        "url": "......"
        "hash": "......"
    }
}
```

*pelook* result:
```
> pelook "F:\mtpcopy.exe"
loaded "F:\mtpcopy.exe" / 1569280 (0x17F200) bytes
signature/type:       PE64 EXE image for amd64
image checksum:       0x00000000 (calc=0x0018E45D)
machine:              0x8664 (amd64)
subsystem:            3 (Windows Console)
minimum os:           6.0 (Vista)
linkver:              Microsoft LINK 14.29
detected toolset(s):  Visual C++ 9.0 2008 SP1 (build 30729) <--COMPILER(s)
```

* Add **checkver** and **autoupdate** so that *Scoop* can track the updates of this app. For example:
```
"checkver": "github",
"autoupdate": {
    "architecture": {
        "64bit": {
            "url": "https://github.com/kzmi/mtpcopy/releases/download/$version/mtpcopy.exe"
        }
    }
}
```

You can check existing packages, or [our Wiki](https://github.com/ScoopInstaller/Scoop/wiki/App-Manifests) for more information.

**Comment by animegamer4422** [comment]
Sorry I'm very new to I'll try to do it properly in some time

**Comment by issaclin32** [comment]
> Sorry I'm very new to this , I'll try to do it properly in some time

It's fine. No worries.

File: bucket/mtpcopy.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "homepage": "https://github.com/kzmi/mtpcopy",
+    "description": "A Windows CLI tool for copying files from/to a portable device via MTP.",
+    "license": "GNU",
+    "version": "0.3.0",
+    "url": "https://github.com/kzmi/mtpcopy/releases/download/0.3.0/mtpcopy.exe",
+    "bin": "mtpcopy.exe",
+    "hash": "25768b655afcb8ef1ec9afb5d775c6b73ca44f9b8d35297a72cc94b4638910f8"
+}
```
Comment (issaclin32) [inline bucket/mtpcopy.json:9]:
```suggestion
{
    "version": "0.3.0",
    "description": "A Windows CLI tool for copying files from/to a portable device via MTP",
    "homepage": "https://github.com/kzmi/mtpcopy",
    "license": "MIT",
    "architecture": {
        "64bit": {
            "url": "https://github.com/kzmi/mtpcopy/releases/download/0.3.0/mtpcopy.exe",
            "hash": "25768b655afcb8ef1ec9afb5d775c6b73ca44f9b8d35297a72cc94b4638910f8"
        }
    },
    "bin": "mtpcopy.exe",
    "checkver": "github",
    "autoupdate": {
        "architecture": {
            "64bit": {
                "url": "https://github.com/kzmi/mtpcopy/releases/download/$version/mtpcopy.exe"
            }
        }
    }
}
```

**Comment by issaclin32** [comment]
@animegamer4422 If you want this package to be added to *Scoop* now, you can just click "commit suggestion" to apply the suggestion.

File: bucket/mtpcopy.json (hunk 1 above)
Comment (animegamer4422) [reply bucket/mtpcopy.json]: I've added your code in the file , anything else i can do that will make it better?

Comment (issaclin32) [reply bucket/mtpcopy.json]: > I've added your code in the file , anything else i can do that will make it better?

Nothing else. I think we have all the items needed.

**Pull request merged**